activesupport/lib/active_support/time_with_zone.rb

```diff
@@ -358,11 +364,14 @@ def to_r
       utc.to_r
     end
 
-    # Return an instance of Time in the system timezone.
+    # Returns an instance of Time in the system timezone.
     def to_time
       utc.to_time
     end
 
+    # Returns an instance of DateTime in the UTC timezone.
+    #
+    #   Time.zone.now.to_datetime   # => Tue, 18 Aug 2015 02:32:20 +0000
```

Comment:
@ronakjangir47 this comment is incorrect - it's the AS::TWZ offset that's used not UTC, e.g:

``` irb
>> Time.current.in_time_zone('Hawaii').to_datetime
=> Tue, 25 Aug 2015 09:51:12 -1000
```

Can you create a new PR with the correction - thanks!